dvc/output/base.py

```diff
@@ -231,3 +241,7 @@ def get_files_number(self):
             return 1
 
         return 0
+
+    def unprotect(self):
+        if self.exists:
+            self.remote.unprotect(self.path)
```

Comment:
Should be
```suggestion
            self.remote.unprotect(self.path_info)
```
with all the logical conequenses 🙂